editdictionaries.cc

```diff
@@ -271,12 +271,19 @@ void EditDictionaries::helpRequested()
 
     if( helpWindow )
     {
-      helpWindow->setWindowFlags( Qt::Window );
+      #ifdef Q_OS_MAC
+        helpWindow->setWindowFlags( Qt::Dialog );
+      #else
+        helpWindow->setWindowFlags( Qt::Window );
+      #endif
```

Comment:
This could be simplified (and properly reformatted because the preprocessor directives are normally not indented and the indentation of code within is usually unaffected by conditional compilation) as:
```cpp
#ifndef Q_OS_MAC
      helpWindow->setWindowFlags( Qt::Window );
#endif
```

That's because the `Qt::Dialog` flag is set by default [here](https://codebrowser.dev/qt5/qtbase/src/widgets/dialogs/qdialog.cpp.html#_ZN7QDialogC1EP7QWidget6QFlagsIN2Qt10WindowTypeEE) (same in Qt 6).

Reply:
> This could be simplified

Thanks for the good advice. The second option is more beautiful and concise, but the first one is more human-readable.

> properly reformatted

My apologies. I saw that directives are not indented in the code, but for some reason I didn't pay much attention to it. I will be more careful next time.

Reply:
> but the first one is more human-readable.

Arguable, but as the code is more readable to you, let's keep it as is.